azure-pipelines.yml

```diff
@@ -48,10 +48,10 @@ jobs:
 
 - job: BuildArtifacts_Windows
   dependsOn: InitializePipeline
-  condition: succeeded()
   variables:
     bypassPackaging: $[ dependencies.InitializePipeline.outputs['Initialize.BypassPackaging'] ]
     includeSuffix: $[ dependencies.InitializePipeline.outputs['Initialize.IncludeSuffix'] ] 
+  condition: and(succeeded(), eq(variables['bypassPackaging'], false))
```

Comment:
let's add this same thing to BuildArtifacts_Ubuntu?

Reply:
But none of these tests run in a linux environment. So thought it would be good to keep BuildArtifacts_ubuntu as is..